recipe/meta.yaml

```diff
@@ -1,42 +1,44 @@
 {% set name = "pyoptsparse" %}
-{% set version = "2.8.3" %}
+{% set version = "2.9.0" %}
```

Comment:
Oh I thought I saw the version got bumped, that explains the lack of a new tag

Reply:
Yeah the version/hash should be kept as it is in the main branch, and the bot will automatically create PRs with those changes as we make new releases upstream.

Reply:
Wait so you're saying totally revert the version and has back to what's in the main branch of this repo and conda-forge will automatically change the version to a new tag? Cause as it is this will pull down v2.8.3 which doesn't point to the meson changes

Reply:
Yes, my understanding is that this PR should not trigger a build in the feedstock, but I will make a new release upstream as soon as this is merged, and that should then trigger a new build (via a PR where a bot will bump the version/hash) which will use the new meson-based build process.

Reply:
Okay seems like it's all in. Sorry, I wasn't understanding how it all worked